### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by dariia-romanova**
[DEMO LINK](https://dariia-romanova.github.io/js_task_fix_form_DOM/)

### CODE REVIEW

**Review by SerhiiKirik — APPROVED** [review]
Very nice 🔥

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write code here
+function convertCamelCase(text) {
+  return text.replace(/([A-Z])/g, ' $1');
```
Comment (AlinaMatuschak) [inline src/scripts/main.js:4]: 🔥

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+
+for (const input of [...inputs]) {
+  const label = document.createElement('label');
+
+  label.className = 'field-label';
+  label.htmlFor = input.id;
+  label.textContent = convertCamelCase(input.name);
+  input.parentElement.append(label);
+
+  input.placeholder = convertCamelCase(input.name).toUpperCase();
```
Comment (AlinaMatuschak) [inline src/scripts/main.js:17]: you can create variable for convertCamelCase(input.name)

**Review by AlinaMatuschak — APPROVED** [review]
🔥

**Pull request closed without merging**